plumex/post/edge_figs.py

```diff
@@ -0,0 +1,334 @@
+    "wind": {"default": ("61645e", "blender-wind", "a559a7")},
+    "low 1": {"default": ("677028", "low-1", "ea256d")},
+    "hi 1": {"default": ("729d03", "hi-1", "487513")},
+    "hi 2": {"default": ("56a332", "hi-2", "3115a5")},
+}
+
+
+vids = [
+    "low 862",
+    "low 865",
+    "low 867",
+    "low 869",
+    "low 913",
+    "med 871",
+    "med 914",
+    "med 916",
+    "hi 919",
+    "hi 920",
+]
+
+
+def _unpack_data(
+    center_regress_hash: str,
+    video_lookup_keyword: str,
+    edge_regress_hash: str,
+) -> RegressionData:
+
+    video, orig_center_fc = _load_video(data_lookup["filename"][video_lookup_keyword])
+    center_mitosis_step = mitosis.load_trial_data(
+        hexstr=center_regress_hash, trials_folder=trials_folder / "center-regress"
+    )
+    edge_mitosis_step = mitosis.load_trial_data(
+        hexstr=edge_regress_hash, trials_folder=trials_folder / "edge-regress"
+    )
+
+    center_fit_method = center_mitosis_step[1]["main"]
+    center_coeff_dc = center_mitosis_step[1]["regressions"][center_fit_method]["data"]
+
+    edge_plume_points = edge_mitosis_step[0]["data"]
+
+    center = cast(List[tuple[int, PlumePoints]], edge_plume_points["center"])
+    bot = cast(List[tuple[int, PlumePoints]], edge_plume_points["bottom"])
+    top = cast(List[tuple[int, PlumePoints]], edge_plume_points["top"])
+
+    top_edge_method, bot_edge_method = edge_mitosis_step[1]["main"]
+
+    edge_coefs_top = np.nanmean(
+        edge_mitosis_step[1]["accs"]["top"][top_edge_method]["coeffs"], axis=0
+    )
+    edge_coefs_bot = np.nanmean(
+        edge_mitosis_step[1]["accs"]["bot"][bot_edge_method]["coeffs"], axis=0
+    )
+
+    top_func = create_edge_func(edge_coefs_top, top_edge_method)
+    bot_func = create_edge_func(edge_coefs_bot, bot_edge_method)
+
+    start_frame = center[0][0]
+    end_frame = center[-1][0]
+
+    return RegressionData(
+        video=video[start_frame : end_frame + 1],
+        center_coef=center_coeff_dc,
+        center_func_method=center_fit_method,
+        center_plume_points=center,
+        top_plume_points=top,
+        bottom_plume_points=bot,
+        top_edge_func=top_func,
+        bot_edge_func=bot_func,
+        start_frame=start_frame,
+        orig_center_fc=orig_center_fc,
+    )
+
+
+vid_names = ["low 869", "med 914", "hi 920"]
+frame_ids = [250, 750, 1000, 1200]
+
+
+video_data = [_unpack_data(*trial_lookup_key[name]["default"]) for name in vid_names]
+
+
+def _create_fig2b_raw():
+
+    figsize = (20, 10)
+    what_to_plot = {
+        "plot_edge_points": True,
+        "plot_center_points": False,
+        "plot_edge_regression": True,
+        "plot_center_regression": False,
+    }
+
+    _visualize_multi_edge_fits(
+        video_data=video_data,
+        frame_ids=frame_ids,
+        title="Edge Regression (Raw Points)",
+        subtitles=vid_names,
+        figsize=figsize,
+        plot_on_raw_points=True,
+        **what_to_plot,
+    )
```

Comment:
If you're creating the dict to unpack, why not just define call `_visualize...` with all the arguments?